homeassistant/components/ld2410_ble/translations/ca.json

```diff
@@ -0,0 +1,23 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "El dispositiu ja est\u00e0 configurat",
+            "already_in_progress": "El flux de configuraci\u00f3 ja est\u00e0 en curs",
+            "no_devices_found": "No s'han trobat dispositius a la xarxa",
+            "no_unconfigured_devices": "No s'han trobat dispositius no configurats.",
+            "not_supported": "Dispositiu no compatible"
+        },
+        "error": {
+            "cannot_connect": "Ha fallat la connexi\u00f3",
```

Comment:
Drop any languages except en.json since they will get overwritten by lokalize anyways

Reply:
That's neat. Done.